### PR TITLE
feat: add latest block metrics

### PR DESCRIPTION
**Pull request by parketh**
## Summary

This PR:
1. adds a new stats component to display the latest block info
2. applies some formatting / state mgmt changes

### `Stats` component

We add a new `Stats` component to display the latest block info, including:

- Latest L2 block
- Latest BTC finalized L2 block
- Earliest consecutively BTC finalized L2 Block
- Latest ETH finalized L2 block

The data is fetched from the new finality gadget api route implemented at https://github.com/babylonlabs-io/finality-gadget/pull/31

### CSS / state management updates

- Updated primary color to match Snapchain convention
- Improve loading state management for tx info component

## Test plan

```
npm run dev
```

image

### CODE REVIEW

**Comment by bap2pecs** [comment]
how did you test this given the chain is not up? can you add some screenshots in the Test Plan section?

File: src/app/page.tsx [hunk 1]
```diff
@@ -66,16 +85,31 @@ const Home: React.FC<HomeProps> = () => {
...
   return (
     <main className={`relative h-full min-h-svh w-full main-app-background`}>
       <Header />
       <div className="container mx-auto flex justify-center p-6">
         <div className="container flex flex-col gap-6">
+          <Stats latestBlockInfo={latestBlockInfo} />
           <SearchBar searchTerm={searchTerm} setSearchTerm={setSearchTerm} />
-          {txInfo && !isLoadingTxInfo ? (
-            <Transaction transaction={txInfo} />
+          {!!searchTerm ? (
```
Comment (bap2pecs) [inline src/app/page.tsx:111]: why is `txInfo` changed to `!!searchTerm`?

File: src/app/assets/gold-check.svg
[the remark this answers is not in the document]
Comment (parketh) [reply src/app/assets/gold-check.svg:1]: now removed

File: src/app/page.tsx (hunk 1 above)
Comment (parketh) [reply src/app/page.tsx:111]: the loading state is handled inside the <Transaction> component. We want the component to render from the moment user types an input, not when the tx info has finished loading

Comment (bap2pecs) [reply src/app/page.tsx:111]: ah makes sense. let's marge the PR this time :)

**Review by bap2pecs — APPROVED** [review]

**Pull request merged**